packages/core/src/utils/prepareEvent.ts

```diff
@@ -114,6 +114,8 @@ function applyClientOptions(event: Event, options: ClientOptions): void {
   }
 }
 
+const debugIdStackParserCache = new Map<StackParser, Map<string, StackFrame[]>>();
```

Comment:
Should we use `WeakMap` to ensure garbage collection can still work? Or will this mess the cache up 🤔

Reply:
We can!

Reply:
Did think about this but I thought weak map wasn't a thing in ie11 - turns out it is.